bokehjs/src/lib/core/dom_view.ts

```diff
@@ -53,6 +53,11 @@ export abstract class DOMView extends View {
 
   abstract render(): void
 
+  rerender(): void {
+    this.render()
+    this.r_after_render()
```

Comment:
What does the "r" designate?

Reply:
`r` stands for `recursive`, though this isn't currently a consistent naming scheme. E.g. `update_layout()` is recursive, but not indicated as such. Typically `r_` methods will appear where there's already a non-recursive method of the same name.